README.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal Potfolio
+
+> This project is about Dameh Seko as a web developer. I am very passoinate about software. Get in touch with me through any of my handles listed below and i will be of help. See the technologies i can use below. 
+
+## Built With
+
+- HTML
+- CSS
+- Microverse linters
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
```

Comment:
As said by the previous reviewer, Kindly make sure you customize the README file to this specific project. Please, fill the sections with instructions and information about this project